libc/utils/docgen/cpio.yaml

```diff
@@ -0,0 +1,107 @@
+macros:
+  C_IRUSR:
+    c-definition: "Read by owner."
```

Comment:
The `c-definition` field is being used for "what section in the C standard documents this macro/funcion." In the case of cpio.h, it's POSIX only and not standard C. So the `c-definition` field can be removed for each entry.